modules/nf-core/phantompeakqualtools/tests/main.nf.test

```diff
@@ -0,0 +1,88 @@
+// nf-core modules test phantompeakqualtools
+nextflow_process {
+
+    name "Test Process PHANTOMPEAKQUALTOOLS"
+    script "../main.nf"
+    process "PHANTOMPEAKQUALTOOLS"
+
+    tag "modules"
+    tag "modules_nfcore"
+    tag "phantompeakqualtools"
+
+    test("sarscov2 - bam - single_end") {
+
+        when {
+            process {
+                """
+                input[0] = [
+                    [ id:'test', single_end:false ], // meta map
+                    file(params.test_data['sarscov2']['illumina']['test_paired_end_bam'], checkIfExists: true)
+                ]
+                """
+            }
+        }
+
+        then {
+            assertAll(
+                { assert process.success },
+                { assert snapshot(process.out.spp[0][1]).match() },
+                { assert snapshot(
+                    process.out.uppercase.collect { it.collect { it instanceof String && file(it).exists() ? file(it).name : it }}
```

Comment:
I don't know what this is trying to do.

Reply:
I took it from [here](https://github.com/nvnieuwk/nf-test-snippets/tree/main?tab=readme-ov-file) "Output everything from the process/workflow without checking md5sum of the files - only check existence"